build.fsx

```diff
@@ -252,6 +252,7 @@ let dotnetAssemblyInfos =
       "Fake.BuildServer.Travis", "Integration into Travis buildserver"
       "Fake.Core.CommandLineParsing", "Core commandline parsing support via docopt like syntax"
       "Fake.Core.Context", "Core Context Infrastructure"
+      "Fake.Core.DependencyManager.Paket", "Paket Dependency Manager"
```

Comment:
(Remainder) Maybe this should be named `Fake.Runtime.PaketDependencyManager` instead.
But as we have discussed this is probably temporary (and this is basically implementation detail, so we probably can change that anytime)
/cc @baronfel

Reply:
along with this, the dependencymanager build has warnings that get logged that caused FSAC to throw errors initially (because FSAC has WarningsAsErrors turned on).

```
/Users/chethusk/oss/FAKE/src/app/Fake.Runtime/unknown(1,1): warning FS3186: An error occurred while reading the F# metadata node at position 1 in table 'itycons' of assembly 'Fake.Core.DependencyManager.Paket, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The node had no matching declaration. Please report this warning. You may need to recompile the F# assembly you are using. [/Users/chethusk/oss/FAKE/src/app/Fake.Runtime/Fake.Runtime.fsproj]
/Users/chethusk/oss/FAKE/src/app/Fake.Runtime/unknown(1,1): warning FS3186: An error occurred while reading the F# metadata node at position 0 in table 'ivals' of assembly 'Fake.Core.DependencyManager.Paket, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'. The node had no matching declaration. Please report this warning. You may need to recompile the F# assembly you are using. [/Users/chethusk/oss/FAKE/src/app/Fake.Runtime/Fake.Runtime.fsproj]
```

these are...fairly interesting, because they related to the pickled metadata nodes. I don't know which nodes in question are failing though.

Reply:
Interestingly, it seems like `release/next` is now red as well.

Reply:
It looks like there is still some underlying issue here. I have no idea what the problem is.

Reply:
@baronfel is there some naming scheme we need to follow for FCS/F# Compiler to find the dependency-manager? I guess my suggestion doesn't follow that.

Reply:
Yes, I added a comment to the fsproj for the dependency manager with the pattern the compiler looks for. It looks for *DependencyManager*.dll for candidates.